perf/Microsoft.AspNetCore.Server.Kestrel.Performance/configs/CoreConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using BenchmarkDotNet.Configs;
+using BenchmarkDotNet.Engines;
+using BenchmarkDotNet.Environments;
+using BenchmarkDotNet.Jobs;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Performance
+{
+    public class CoreConfig : ManualConfig
+    {
+        public CoreConfig()
+        {
+            Add(Job.Default.
```

Comment:
Rather than calculating the 'RPS' by hand, you can use the handy `RpsColumn` class I put together for you!! See [this gist](https://gist.github.com/mattwarren/b24ffcde925979d10ee80a3929d7143d) for the code

Also I always find it good to put `Add(JitOptimizationsValidator.FailOnError);` into the config, as it will stop you from accidentally running a benchmark in Debug more!! It's [included in the DefaultConfig](https://github.com/dotnet/BenchmarkDotNet/blob/master/src/BenchmarkDotNet.Core/Configs/DefaultConfig.cs#L48-L53) but not if you make your own.